services/web/client/source/class/osparc/dashboard/OrganizationEditor.js

```diff
@@ -0,0 +1,162 @@
+/* ************************************************************************
+
+   osparc - the simcore frontend
+
+   https://osparc.io
+
+   Copyright:
+     2020 IT'IS Foundation, https://itis.swiss
+
+   License:
+     MIT: https://opensource.org/licenses/MIT
+
+   Authors:
+     * Odei Maiz (odeimaiz)
+
+************************************************************************ */
+
+qx.Class.define("osparc.dashboard.OrganizationEditor", {
```

Comment:
what about ``JsonSchemaForm``...? let's reuse it!

Reply:
it's the perfect opportunity to make it really useful. this guy has validation for free... and no need to write more code for each new form we want... just define it with jsonschema (easy to change too)

Reply:
Will take it into account for the next form 👍